### PR TITLE
fix: navigation to user profile page

### LINKED ISSUE #162: Navigate to `my.realdevsquad.com`
If someone clicks on `Hello, <username> and profile picture`, the user should be taken to https://my.realdevsquad.com/profile
image

### PR DESCRIPTION
**Pull request by rajat-mehra05**
Closes #162 
1. Added link to redirect user to `https://my.realdevsquad.com/profile` page on clicking `Hello, user-name`
2. Replaced `realdevsquad.com` with `www.realdevsquad.com` in `events.html` and `releases.html` files
3. Fixed `https://crypto.realdevsquad.com/auction` link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,10 +61,15 @@
...
-          <div class="user-greet-msg">Hello, User!</div>
-          <img class="user-profile-pic" />
-        </div>
+
+        
+        <div class="user-greet">        
+            <a href="https://my.realdevsquad.com/profile">
+            <div class="user-greet-msg">Hello, User!</div>
+            <img class="user-profile-pic" />      
+          </a>
```
Comment (rohan09-raj) [inline index.html:70]: Please fix the formatting here.
```
<a href="https://my.realdevsquad.com/profile">
  <div class="user-greet-msg">Hello, User!</div>
  <img class="user-profile-pic" />      
</a>
```
It should be like this.

File: css/navbar.css [hunk 2]
```diff
@@ -65,6 +65,16 @@ nav li a:hover {
   margin: 25px;
 }
 
+.user-greet a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.user-greet a:hover {
+  color: #49a82e;
```
Comment (rohan09-raj) [inline css/navbar.css:74]: Why are we changing the color on hover ??

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
@rajat-mehra05 
1- Please check there are two login buttons in the `index.html` so the thing is we are using two buttons for login, one for mobile view and one for desktop view. That's why you were not able to click on the user greet when the screen size was greater than `970px`.
2- Also, please add the anchor tags to user-greet in `releases.html` and `events.html`, as these pages are also part of this site and they to have to navbar similar to `index.html`

File: css/navbar.css (hunk 2 above)
Comment (rajat-mehra05) [reply css/navbar.css:74]: The other buttons on navbar seem to have the same property, that's why I had put `hover` property there.

Comment (rohan09-raj) [reply css/navbar.css:74]: No for this button we don't need any color change, only cursor changing to `pointer` will do the job.

**Review by rohan09-raj — APPROVED** [review]
Looks great to me. Approving the changes, thanks for sticking around.

**Pull request merged**